src/components/picture/index.stories.mdx

```diff
@@ -0,0 +1,147 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Picture from ".";
+import Source from "./_children/source";
```

Comment:
This shouldn't be needed as you should never import the children components

Reply:
Thanks! Removed.

Reply:
I see this marked as resolved - but the code is still here